main_test.go

```diff
@@ -1,20 +1,168 @@
 package main
 
 import (
-	"os"
+	"context"
+	"database/sql"
 	"testing"
-	"text/tabwriter"
+	"time"
 
 	"github.com/urfave/cli/v2"
 	"gotest.tools/v3/assert"
 )
 
-func TestExample(t *testing.T) {
-	t.Log("Test Example")
+func tearDownCustomers() error {
+	db, err := connectDB()
+	if err != nil {
+		return err
+	}
+
+	_, err = db.Exec("DELETE FROM Customers")
+	if err != nil {
+		return err
+	}
+	_, err = db.Exec("ALTER TABLE Customers AUTO_INCREMENT=1")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func tearDownProducts() error {
+	db, err := connectDB()
+	if err != nil {
+		return err
+	}
+
+	_, err = db.Exec("DELETE FROM Products")
+	if err != nil {
+		return err
+	}
+	_, err = db.Exec("ALTER TABLE Products AUTO_INCREMENT=1")
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func tearDownOrders() error {
+
+	db, err := connectDB()
+	if err != nil {
+		return err
+	}
+
+	_, err = db.Exec("DELETE FROM Orders")
+	if err != nil {
+		return err
+	}
+	_, err = db.Exec("ALTER TABLE Orders AUTO_INCREMENT=1")
+	if err != nil {
+		return err
+	}
+	err = tearDownCustomers()
+	if err != nil {
+		return err
+	}
+	err = tearDownProducts()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func createCustomersData(args [][]string) error {
+	db, err := connectDB()
+	if err != nil {
+		return err
+	}
+
+	app := &cli.App{
+		Commands: []*cli.Command{
+			newCreateCustomerCommand(db),
+		},
+	}
+
+	for _, val := range args {
+		err = app.Run(val)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
 }
 
-func TestExample2(t *testing.T) {
-	t.Log("Test Example 2")
+func createCustomersDataV2(args [][]string) {
+
+	db, _ := connectDB()
```

Comment:
Make sure to always check your errors, otherwise this just gets more difficult debug when things go wrong. When testing, you can pass in a `*testing.T` for faster error handling:
```go
func createCustomersDataV2(t *testing.T, args [][]string) {
  db, err := connectDB()
  if err != nil {
    t.Fatal(err)
  }
    ...
}
```